backend/internal/service/handler/lineItem/batch_update_scope_emissions.go

```diff
@@ -0,0 +1,46 @@
+package lineItem
+
+import (
+	"log"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+type UpdateLineItemsRequest struct {
+	LineItemIDs       []uuid.UUID `json:"line_item_ids"`
+	Scope             *int        `json:"scope,omitempty"`
+	EmissionsFactorID *string     `json:"emissions_factor_id,omitempty"`
+}
+
+func (h *Handler) BatchUpdateLineItems(c *fiber.Ctx) error {
+	var req UpdateLineItemsRequest
+
+	if err := c.BodyParser(&req); err != nil {
+		log.Println("Error parsing request body:", err)
+		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"error": "Invalid request payload"})
```

Comment:
wait lol i meant to leave this comment and think i forgot to save it whoops

but anyway I think we can replace this with our centralized error handling methods for consistency and to slightly simplify the code. (it'll automatically print the error message for you)



```suggestion
		return errs.BadRequest("Invalid request payload: " + err.Error())

```

<img width="621" alt="image" src="https://github.com/user-attachments/assets/f7013c1b-d770-4f7e-b653-f1f4f6c1699c" />